Frontend/src/test/pages/cart/components/NeedHelp.test.jsx

```
@@ -0,0 +1,46 @@
+import { render, screen, fireEvent, cleanup } from '@testing-library/react';
+import { vi } from 'vitest';
+import NeedHelp from '../../../../pages/cart/components/NeedHelp';
+
+// Mock UI components
+vi.mock('../../../../components/ui/card', () => ({
+  Card: ({ children }) => <div data-testid="Card">{children}</div>,
+  CardContent: ({ children }) => <div data-testid="CardContent">{children}</div>
+}));
+vi.mock('../../../../components/ui/button', () => ({
+  Button: ({ children, onClick }) => (
+    <button data-testid="Button" onClick={onClick}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)